custom-namespaces.yaml

```diff
@@ -714,3 +714,26 @@ shared:
       type: table_view
       tables:
         - table: mozdata.static.data_incidents_v1
+site:
+  glean_app: false
+  pretty_name: Site
```

Comment:
Lets change pretty name to `Websites`

Reply:
Changes it to `Mozilla Org Websites` to make it explicit that this namespace related to Mozilla's websites